gen/crm_ceitec

```diff
@@ -38,16 +37,8 @@ my $userGroups; # $userGroups->{$login}->{$groupName} = 1;
 # FOR EACH RESOURCE
 foreach my $resourceId ( $data->getResourceIds() ) {
 	foreach my $groupId ( $data->getGroupIdsForResource( resource => $resourceId ) ) {
-
+		# EXPIRED members are not sent from Perun
```

Comment:
You can update the script version and I think we should omit this comment, because if someone decides to reconfigure the service in gui, it won't be updated in this here.